.github/workflows/package.yml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     tags:
       - '*'
+  pull_request:
+    paths:
+      - '.github/workflows/package.yml'
```

Comment:
Add comment
# Also run this workflow when this package.yml is updated by a PR

Reply:
Fixed with a35f531